api/modelss/build-log.js

```diff
@@ -0,0 +1,24 @@
+const associate = ({ BuildLog, Build }) => {
+  BuildLog.belongsTo(Build, {
+    foreignKey: "build",
+    allowNull: false,
+  })
+}
+
+module.exports = (sequelize, DataTypes) => {
+  const BuildLog = sequelize.define("BuildLog", {
+    output: {
+      type: DataTypes.STRING,
+    },
+    source: {
+      type: DataTypes.STRING,
+    },
+  }, {
+    tableName: "buildlog",
+    classMethods: {
+      associate,
```

Comment:
When you setup these associations, you need all of the models defined. This does not work when the modules are described in multiple files because you end up with circular requires and models that are being associated before they are defined.

There is an [issue for this](https://github.com/sequelize/sequelize/issues/4439) and [supposedly the team is working on a fix for the next version](https://github.com/sequelize/sequelize/issues/6524). In the meantime, the suggestion in the open issue is the follow [this pattern which is laid out in the Express sample project](https://github.com/sequelize/express-example/blob/master/models/task.js#L8-L16).